pod/perldiag.pod

```diff
@@ -1042,7 +1042,7 @@ error occurs in cases such as these:
     goto G;
     $x + do { G: $y }; # How is + supposed to get its first operand?
 
-=item Can't "goto" into a defer block
+=item Can't "goto" into a "defer" block
 
 (F) A "goto" statement was executed to jump into the scope of a defer
```

Comment:
`C<defer>` here too (and maybe `C<goto>` instead of `"goto"`)?

I don't mean to turn this into an audit of all references to keywords in the document, but while we're nearby …

Reply:
Done in 54cf2adaef